app/assets/javascripts/components/rsvp_show.es6

```diff
@@ -24,18 +27,30 @@ const RsvpShow = React.createClass({
     return rsvpRows
   },
 
+  renderEventHeaders(rsvps) {
+    let rsvp_headers = _.map(rsvps, (rsvp) => {
+      return (
+       <div key={rsvp.id}>
```

Comment:
This looks like it's one space to the right of return instead of a full tab, same with the line below. Can you update the tabbing for Lines 33-39?